<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShapeWindow</class>
 <widget class="QMainWindow" name="ShapeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1008</width>
    <height>644</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ShapeWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>10</y>
      <width>352</width>
      <height>491</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="addRemoveInfo">
       <property name="text">
        <string>ADD / REMOVE FORM: </string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="allShapes">
       <item>
        <property name="text">
         <string>Shapes</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Polygon</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Polyline</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Rectangle</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Ellipse</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Line</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="existShapeInfo">
       <property name="text">
        <string>Exsiting Shape:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="selectedShapes"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="addInfo">
       <property name="text">
        <string>Clike this Button to ADD this shape -&gt;</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>ADD</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="deleteInfo">
       <property name="text">
        <string>Click this Button to DELETE this shape -&gt;</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>DELETE</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout"/>
     </item>
     <item row="4" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="shapeName">
         <property name="text">
          <string>   Shape</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="addOrDelete">
         <property name="text">
          <string>ADD? DELETE?</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="shapeEditor">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>310</width>
      <height>530</height>
     </rect>
    </property>
    <property name="title">
     <string>Shape Editor</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>45</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>ID:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="shapeComboBox">
     <property name="geometry">
      <rect>
       <x>101</x>
       <y>40</y>
       <width>191</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLCDNumber" name="idLCD">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>40</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="digitCount">
      <number>2</number>
     </property>
    </widget>
    <widget class="QWidget" name="formLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>271</width>
       <height>421</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="shapeForm"/>
    </widget>
   </widget>
   <widget class="QWidget" name="pWidget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>120</width>
      <height>80</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1008</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>addOrDelete</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>862</x>
     <y>145</y>
    </hint>
    <hint type="destinationlabel">
     <x>912</x>
     <y>332</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>addOrDelete</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>862</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>912</x>
     <y>332</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>allShapes</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>shapeName</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>862</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>863</x>
     <y>332</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>allShapes</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>selectedShapes</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>952</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>952</x>
     <y>71</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
